packages/universal-components/src/RangeDatePicker/RenderDay.js

```diff
@@ -42,28 +43,32 @@ export default function RenderDay({
   return (
     <View style={styles.container}>
       <Touchable onPress={handlePress} disabled={isDayInPast || isFieldEmpty}>
```

Comment:
I'm not sure if it is caused by this PR. But on Android inactive days are still clickable.
<img width="221" alt="Screenshot 2019-03-28 at 11 49 23" src="https://user-images.githubusercontent.com/2660330/55151993-bb690900-514f-11e9-95f1-b8b1555f965f.png">

Reply:
Should be fixed now

Reply:
Thanks, it looks good now.